### ISSUE TITLE
Advice for developing a dashboard web interface

### ISSUE THREAD

**Issue by zxl716** [opencurve/curve#2557]
curveFS/FS storage system, it is recommended to develop the dashboard web interface management function

**Comment by ilixiaocui** [comment]
The dashbord of CurveBS has almost been developed. After the test is completed, it will be open sourced. CurveFS welcomes interested partners to develop together.

**Comment by aspirer** [comment]
> 

yes, we will push source codes here in July, and there are many minor bugs in dashboard project, so we need deveplopers to fix them.

**Comment by zxl716** [comment]
Expect Dashboard Web Experience
